### ISSUE TITLE
Eta vs Ejs link invalidity

### ISSUE THREAD

**Issue by atomdevelops** [eta-dev/eta#178]
**Describe the bug**
Inside the [Eta vs ejs](https://eta.js.org/docs/about/eta-vs-ejs), the link (https://ghcdn.rawgit.org/eta-dev/eta/694976ffcfbe5fc2e5ae3f891549849a3bba9fdd/browser-tests/benchmark.html) directs to a spam website. It would be helpful and best if it were replaced with the accurate link, or removed totally.

**To Reproduce**
1. Open the eta website
2. [Go to this link](https://eta.js.org/docs/about/eta-vs-ejs)
3. Look at the link after "Check out these benchmarks:"

**Expected behavior**
Open a page showcasing the benchmarks compared to EJS.

**Package & Environment Details**
 - Environment: Firefox
 - Version: 10.0.1 (64-bit)
 - Operating System: Windows 11

**Comment by shadowtime2000** [comment]
Yeah, the CDN we used to serve benchmark got swapped but I guess we forgot to change it in the docs. If you want to change this yourself, you can do it, otherwise I'll get it done soon. 

Since this has to do with the documentation website, I'm going to transfer the issue there.
